app/assets/stylesheets/utilities/_overwrites.scss

```diff
@@ -78,3 +78,19 @@
     font-size: inherit;
   }
 }
+
+.govuk-link {
+  &.govuk-link--no-decoration {
+    text-decoration: none;
+  }
+}
+
+.govuk-list {
+  .govuk-list-item--with-border-bottom {
```

Comment:
Similar here, we shouldn't nest them.

These two in particular look quite ugly, especially the --font-size-16. I would push this work to be part of the side navigation component instead as per https://trello.com/c/ZhPzVCUr